legend-depot-server/src/main/java/org/finos/legend/depot/server/resources/ProjectsVersionsResource.java

```diff
@@ -58,7 +59,17 @@ public Optional<ProjectVersionDTO> getProjectVersion(@PathParam("groupId") Strin
     {
         return handle(ResourceLoggingAndTracing.GET_PROJECT_VERSION_BY_GAV, ResourceLoggingAndTracing.GET_PROJECT_VERSION_BY_GAV + groupId + artifactId + versionId, () ->
         {
```

Comment:
can we have a separate api pls instead of  mixing this one. So leave existing as is and have a new one
it should be called @Path("/projects/{groupId}/{artifactId}/versions/latest/data")

Reply:
we might have to change the path , otherwise there would be a clash with this API and the existing one

Reply:
there is no clash currently we have latest for revision ie : /metadata/api/projects/{groupId}/{artifactId}/**revisions**/latest 

this wont interfere with the above